src/components/ITSBStore/utils/likelihoodUtils.js

```diff
@@ -54,6 +69,8 @@ export const estimateInterval = (waypoint, graph) => {
       return { waypoint, start, end, likelihood: 3 };
     }
   } else {
+    // Interval 'inference' (as described in the comment above) happens
+    // here - see method below
```

Comment:
Can we move this comment to JSDoc for `inferInterval`? (something like "Helper method to compute the interval inferred according to the process as outlined in the documentation for `estimateInterval`")

Reply:
Done!